pkg/virt-handler/hotplug-disk/mount_test.go

```diff
@@ -61,1057 +64,936 @@ var (
-			return []byte("6,6,0777"), nil
-		}
-		Expect(m.isBlockFile(testFileName)).To(BeFalse())
-		statCommand = func(fileName string) ([]byte, error) {
-			Expect(testFileName).To(Equal(fileName))
-			return []byte("6,6,0777,block special"), nil
-		}
-		Expect(m.isBlockFile(testFileName)).To(BeFalse())
-	})
+			By("Unmounting, we verify the reverse process happens")
+			expectCgroupRule(devices.BlockDevice, 6, 6, false)
+			err = m.unmountBlockHotplugVolumes(filepath.Join(targetPodPath, "testvolume"), vmi)
+			Expect(err).ToNot(HaveOccurred())
+		})
 
-	table.DescribeTable("Should return proper values", func(stat func(fileName string) ([]byte, error), major, minor int, perm string, expectErr bool) {
-		testFileName := "test-file"
-		statCommand = stat
-		majorRes, minorRes, permRes, err := m.getBlockFileMajorMinor(testFileName)
-		if expectErr {
+		It("getSourceMajorMinor should return an error if no uid", func() {
+			vmi.UID = ""
+			_, _, _, err := m.getSourceMajorMinor("fghij", "invalid")
 			Expect(err).To(HaveOccurred())
-		} else {
-			Expect(err).ToNot(HaveOccurred())
-		}
-		// Values are translated to hex (245->580, 32->50)
-		Expect(int64(major)).To(Equal(majorRes))
-		Expect(int64(minor)).To(Equal(minorRes))
-		Expect(perm).To(Equal(permRes))
-	},
-		table.Entry("Should return values if stat command successful", func(fileName string) ([]byte, error) {
-			return []byte("245,32,0664,block special file"), nil
-		}, 581, 50, "0664", false),
-		table.Entry("Should not return values if stat command errors", func(fileName string) ([]byte, error) {
-			return []byte("245,32,0664,block special file"), fmt.Errorf("Error")
-		}, -1, -1, "", true),
-		table.Entry("Should not return values if stat command doesn't return 4 fields", func(fileName string) ([]byte, error) {
-			return []byte("245,32,0664"), nil
-		}, -1, -1, "", true),
-		table.Entry("Should not return values if stat command doesn't return block special file", func(fileName string) ([]byte, error) {
-			return []byte("245,32,0664, block file"), nil
-		}, -1, -1, "", true),
-		table.Entry("Should not return values if stat command doesn't int major", func(fileName string) ([]byte, error) {
-			return []byte("kk,32,0664,block special file"), nil
-		}, -1, -1, "", true),
-		table.Entry("Should not return values if stat command doesn't int minor", func(fileName string) ([]byte, error) {
-			return []byte("254,gg,0664,block special file"), nil
-		}, -1, -1, "", true),
-	)
+		})
 
-	It("getTargetCgroupPath should return cgroup path", func() {
-		slicePath := "slice"
-		expectedCgroupPath := filepath.Join(tempDir, slicePath)
-		m.podIsolationDetector = &mockIsolationDetector{
-			slice: slicePath,
-		}
-		err = os.MkdirAll(expectedCgroupPath, 0755)
-		Expect(err).ToNot(HaveOccurred())
-		path, err := m.getTargetCgroupPath(vmi)
-		Expect(err).ToNot(HaveOccurred())
-		Expect(path).To(Equal(filepath.Join(tempDir, slicePath)))
-	})
+		It("getSourceMajorMinor should return file if exists", func() {
```

Comment:
The test description is a bit unclear. Maybe it would be better to phrase it:
```suggestion
		It("getSourceMajorMinor should succeed if file exists", func() {
```